spec/features/home_spec.rb

```diff
@@ -0,0 +1,24 @@
+feature 'Homepage loads' do
+  
+  scenario "User can't enter a peep if not logged in" do
+    visit '/home'
+    expect(page).to have_no_link('Do a Peep', href: '/create')
+    expect(page).to have_no_button 'Log Out'
+    expect(page).to have_link('Log in', href: '/login')
+  end
+
+  scenario 'User can get to Login' do
+    visit '/home'
+    click_link 'Log in'
+    expect(page).to have_button "log_in"
+  end
+
+  # scenario 'Home displays peeps' do
```

Comment:
I know you probably left this in for future reference, but having large amounts of commented-out code is generally seen as a bad thing.